whipper/common/program.py

```diff
@@ -416,6 +422,9 @@ def getTagList(self, number, mbdiscid):
                     mbidTrack = track.mbid
                     mbidTrackArtist = track.mbidArtist
                     mbidWorks = track.mbidWorks
+                    if classical:
```

Comment:
No need for this check here, I think. It’s fine to assign a `None` value or empty list if `.composers` or `.performers` are not populated.

Reply:
This should also be solved now \o/